WordPress/src/main/java/org/wordpress/android/ui/photopicker/MediaPickerLauncher.kt

```diff
@@ -192,7 +187,8 @@ class MediaPickerLauncher @Inject constructor(
             primaryDataSource = DEVICE,
             availableDataSources = setOf(),
             canMultiselect = canMultiselect,
-            requiresStoragePermissions = true,
+            requiresPhotosVideosPermissions = false,
```

Comment:
❓I believe this is the function that's being called when you navigate from Media  → + Icon  → Choose from Device.  Is the `requiresPhotosVideosPermissions` having correct value here?.  

I did a quick run and test after changing this `requiresPhotosVideosPermissions` to `true`. I was able to see the photos in the Scenario 

- Allowed Photos and Video & Dont allow Audio and Music - I can see the photos and videos
- Dont allow Photos and Video & Allow Audio and Music - I can't see any photos and video, Permission message is shown
- Dont allow both permissions - Permission message is shown

Reply:
Thank you so much for testing and finding a critical problem with the implementation, @AjeshRPai! 
I added the support for "Image picker" or "Audio picker" but not for "Image + video + audio picker".
Now, I have added all media support to the media picker: ac21e3c732fa7d3779a8bb185f1710e3535b5d0c.